spec/services/journal_row_updater_spec.rb

```diff
@@ -0,0 +1,20 @@
+require "rails_helper"
+
+RSpec.describe JournalRowUpdater, type: :service do
```

Comment:
Do we have a spec that checks this when there are multiple journal rows for an order detail? I would guess that spec probably belongs in the split account engine.